docs/source/tutorial/starting.rst

```diff
@@ -56,6 +56,44 @@ install with the command ``make install``.  This will, by default, install into
 
     IDRIS2_PREFIX ?= /usr/local
 
+On Windows
+-------------------------
+
+On Windows systems you can build and use Idris 2 from sources using WSL (Windows Subsystem for Linux).
+
+Open Windows PowerShell as administrator and run:
+
+.. code-block:: PowerShell
+
+	wsl --instal # enable Window to run a linux system 
+	wsl --install -d Ubuntu # install a specific linux distro
+
+A restart may be required.
+
+Open PowerShell and run:
+
+.. code-block:: PowerShell
+
+	wsl -d Ubuntu # open a linux terminal
+	apt update # it is necessary to install dependencies
+	apt install make gcc chezscheme libgmp3-dev # install dependencies required for build from source
+	cd ~ # by default wsl will open in the current Windows directory, so we move to a directory inside the linux filesystem
+	git clone https://github.com/idris-lang/Idris2.git # download source code from github
+	cd Idris2 # move to the source code directory
+	make bootstrap SCHEME=chezscheme # builds the executable
+	make install # installs the executable
+	ln -s ~/.idris2/bin/idris2 /usr/local/bin/idris2 # add the binary to executable path
```

Comment:
You'd need `sudo` (the destination is owned by `root` by default), **however** symlinks are almost always a bad idea (it is very easy to get the order of the arguments wrong, and it creates unnecessary complications in terms of where things are located in this case).

It would be better to instruct users to add the following line at the end of their `~/.bashrc` file and restart their terminal:

```bash
export PATH="$HOME/.idris2/bin:$PATH"
```

(This adds the default Idris2 install location to the magic variable that the shell searches for executables when you try to run a command.)

Reply:
I though about that, but is seems that commands like "wsl -D ubuntu idris2" do not pick up the `~/.bashrc` i will test

Reply:
Any luck? I'm not familiar with that command, I've always just opened up a WSL terminal, and then done everything as I would on a regular linux terminal  : )

Reply:
Yes, at least onmy machine a cannot make it work with bashrc. Anyway i think this feature is not worth the trouble